Tensile/KernelWriterAssembly.py

```diff
@@ -3763,6 +3763,35 @@ def openPersistentLoop(self, kernel):
     if kernel["PersistentKernel"] or kernel["StreamK"]:
       if kernel["StreamK"]:
         # Workload calculations
+        if kernel["StreamKXCCMapping"]:
+          sXCC = self.sgprPool.checkOut(1, "XCC", preventOverflow=0)
+          sGridC = self.sgprPool.checkOut(1, "sGridC", preventOverflow=0)
+          sGridF = self.sgprPool.checkOut(1, "sGridF", preventOverflow=0)
+          sGridM = self.sgprPool.checkOut(1, "sGridM", preventOverflow=0)
+          stmp = self.sgprPool.checkOut(2, "stmp", preventOverflow=0)
+          # sGridC = ceil(grid / xccm)
+          kStr += inst("s_add_u32", sgpr(sXCC), sgpr("skGrid"), 7, "ceil(grid/xccm)")
+          kStr += scalarStaticDivideAndRemainder(sGridC, None, sGridC, kernel["StreamKXCCMapping"], stmp, doRemainder=0)
+          # sGridF = floor(grid / xccm)
+          # sGridM = grid % xccm
+          kStr += scalarStaticDivideAndRemainder(sGridF, sGridM, "skGrid", kernel["StreamKXCCMapping"], stmp)
+          # sXCC = wg0 % xccm
+          kStr += scalarStaticDivideAndRemainder(None, sXCC, "WorkGroup0", kernel["StreamKXCCMapping"], stmp, doRemainder=2)
```

Comment:
Looks like some bugs in scalarStaticDivideAndRemainder.
- qReg==None seems not working if divisor is not power of 2
- not power of 2 cases, this actually uses 3 tmpSgpr (0,1,2) even though the comment says 2 SGPRs

Reply:
Please try XCC not power of 2 case

Reply:
Updated to optionally pass temp quotient register for non-power of 2 case. Works for 8 XCC case. Trying to track down hardware to test other configurations. I'll make a ticket for testing non-power of 2 XCC hardware and if other updates are needed I'll post them in a separate PR.